Azure Sentinel Query Creator.ipynb

```diff
@@ -0,0 +1,1353 @@
+{
```

Comment:
<p>For future...</p><p>You could define a couple of layout/style dicts at the top of this cell - e.g. lbl_lay  = {"layout", widgets.Layout(width='150px')} and then just call Label(..., **lbl_lay)</p><p>Would save a bit of typing.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/82/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>